public/counter/counter.css

```
@@ -0,0 +1,31 @@
+body {
+  display:flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  background-color: #ffffff;
+  font-family: 'Arial', sans-serif;
+}
+.counter-all {
```

```suggestion
}

.counter-all {
```
ファイル毎で空行あり、なしがあったので、空行ありで統一おねがいします！